lib/functions.php

```diff
@@ -378,7 +381,14 @@ function ($matches) {
     if ('///' === substr($uri, 0, 3)) {
         // The triple slash uris are a bit unusual, but we have special handling
         // for them.
-        $result['path'] = substr($uri, 2);
+        /**
+         * We know that this is a string, because the first 3 characters are '///'.
+         * So we can always do a sub-string starting from offset 2 (the 3rd '/').
+         *
+         * @var string $x
```

Comment:
inline `@var` should be prevented when possible.

I wonder why its needed in the first place:
https://phpstan.org/r/e1d7178d-1eff-4a4e-bb0c-cd2d9768c767
-> looks good to me.

Reply:
phpstan kept thinking that it could be `false`. but we know better - we can always do substr from offset 2 at this point.

It seems to understand on PHP 8 but does not understand on PHP 7.4.

Maybe we should run phpstan with PHP 8.? in CI, and not with PHP 7.4. But we still want our code to be compliant with PHP 7.4

Reply:
in 7.4 it can return false. 

please just add a `if ($x === false) throw new \RuntimeException('The string cannot be false');` instead of the inline `@var`

Reply:
done